packages/dagu_car/config/car_cmd_switch_node/default.yaml

```diff
@@ -11,7 +11,7 @@ source_topics: #Mapping from cmd source name to topics
   #coordination: "implicit_coordination_node/car_cmd" #NOTE for implicit coordination
 
   joystick: "joy_mapper_node/car_cmd"
-  vehicle:  "lane_controller_node/car_cmd"
```

Comment:
we really never use the lane_controller_node anymore???

Reply:
We do. The vehicle avoidance node subscribes to lane controller and forwards the same message if no vehicle was detected

Reply:
We do. The vehicle avoidance node subscribes to lane controller and forwards the same message if no vehicle was detected